scripts/inline_comments_data_collection.py

```diff
@@ -158,7 +159,7 @@ def process_comments(patch_threshold, diff_length_threshold):
                 logger.error(f"Failed to fetch diff: {e}")
                 continue
 
-            if len(patch_diff) > diff_length_threshold:
+            if len(patch_diff) > diff_length_threshold and diff_length_threshold != 0:
```

Comment:
I'll be sure to revert these changes. Regarding `diff_length_threshold` and `patch_threshold`, should I implement these as CL arguments rather than specifying their values in `main()`? @suhaibmujahid